doc/api/documentation.md

```diff
@@ -40,9 +40,13 @@ The stability indices are as follows:
 
 <!-- separator -->
 
-> Stability: 3 - Legacy. The feature is no longer recommended for use. While it
-> likely will not be removed, and is still covered by semantic-versioning
-> guarantees, use of the feature should be avoided.
+> Stability 3 - Legacy. Although this feature is unlikely to be removed and is
+> still covered by semantic-versioning guarantees, it is no longer actively
+> maintained, and other alternatives are available.
+
+Features are marked as legacy rather than being deprecated if their use does no
+harm, and they are widely relied upon within the npm ecosystem. Bugs found in
```

Comment:
though not specified anywhere, IMO, the word `ecosystem` is qualified with `Javascript` when we are dealing with a topic that is related to language specification, `Node.js` when the topic is an API spec, and `npm` when it is a module spec. So may be`Node.js ecosystem` is more appropriate here?

Reply:
@gireeshpunathil I chose to use "npm ecosystem" here because it's used in the description of the "stable" stability index: https://github.com/nodejs/node/blob/66db1169d7c9cee01e53217b6f4378df91151d96/doc/api/documentation.md?plain=1#L38-L39
Perhaps we can change both in a separate pr?